lib/profile/view/profile_view.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:very_good_hub/l10n/l10n.dart';
+import 'package:very_good_hub/profile/profile.dart';
+
+class ProfileView extends StatelessWidget {
+  const ProfileView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = context.l10n;
+    final state = context.watch<ProfileBloc>().state;
```

Comment:
Is there a reason for using `context.watch` over a `BlocBuilder` here?

Reply:
No strong reason, just because I need to the whole state, so I can watch the entire bloc, and by using `watch` over `BlocBuilder` I save the code one nesting.